### PR TITLE
[14.0][MIG] product_assortment

### PR DESCRIPTION
**Pull request by xavier-bouquiaux**
tests ok
no changes to module
@rousseldenis

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review.

@xavier-bouquiaux Could you maybe add tests for not covered lines ?

**Comment by xavier-bouquiaux** [comment]
Hi @rousseldenis ,
All line are covered, small changes done in filter

**Review by claudiagn — APPROVED** [review]
LGTM! :+1:

**Comment by rousseldenis** [comment]
@xavier-bouquiaux Could you squash your commits?

**Comment by xavier-bouquiaux** [comment]
> @xavier-bouquiaux Could you squash your commits?

@rousseldenis : Done

File: product_assortment/__manifest__.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Copyright 2018-2019 ACSONE SA/NV
```
Comment (simahawk) [inline product_assortment/__manifest__.py]: copyright should always contain at least the first year. The second is not relevant but 2018 should stay IMO.

**Comment by simahawk** [comment]
@rousseldenis shall we merge? Unless you care about correcting the copyright year :wink:

**Pull request merged**